pkg/util/informermanager/single-cluster-manager.go

```diff
@@ -18,6 +18,9 @@ type SingleClusterInformerManager interface {
 	// The handler should not be nil.
 	ForResource(resource schema.GroupVersionResource, handler cache.ResourceEventHandler)
 
+	// IsInformerSynced
```

Comment:
I'm ok with this interface, even though different from [the one we talked](https://github.com/karmada-io/karmada/pull/479#issuecomment-872674386), but please complement the comments.

Reply:
Let me add it.